skyrim-platform/src/tilted/ui/DX11RenderHandler.cpp

```diff
@@ -131,9 +147,6 @@ void DX11RenderHandler::Create()
 
   m_pStates = std::make_unique<DirectX::CommonStates>(m_pDevice.Get());
 
-  m_pSpriteFont = std::make_unique<DirectX::SpriteFont>(
-    m_pDevice.Get(), L"Data/Platform/Fonts/Tavern.spritefont");
```

Comment:
Let's not delete it and use a `map<string, unique_ptr<SpriteFont>>` to store cached loaded fonts. Otherwise it will cause a performance nightmare where on each frame we'd load font once again for every single text on the screen. In `createText` let's check if we've already loaded this font and if not, then load and put it inside of a map. Btw, have you checked what will happen if the specified font does not exist? Looks like it will cause mysterious crashes